calicoctl/calico_ctl/status.py

```diff
@@ -252,3 +273,42 @@ def pprint_bird_protocols(version):
         print str(x) + "\n"
     else:
         print results + "\n"
+
+
+def _gobgp(cmd):
+    if getattr(sys, 'frozen', False):
+        gobgp = sys._MEIPASS + "/gobgp"
+    else:
+        gobgp = "gobgp"
+    results = subprocess32.check_output([gobgp, "-j"] + cmd.split(" "))
+    return json.loads(results)
+
+
+def pprint_gobgp_protocols(version):
+    x = PrettyTable(["Peer address", "Peer type", "State", "Since", "Info"])
+
+    now = int(time.mktime(datetime.datetime.now().timetuple()))
+    try:
+        neighbors = _gobgp("neighbor -t ipv%d" % version)
+    except subprocess32.CalledProcessError:
```

Comment:
@gunjan5 Could you check `calico-bgp-daemon` is correctly chosen as a backend? I found another bug which should be fixed by [this](https://github.com/projectcalico/calico-containers/pull/1126/commits/689579749de0c2c7e30f0c7d821551e8ad84b082) commit (already added to this PR)

When `calico-bgp-daemon` is running, I couldn't reproduce the issue here.
`calico-bgp-daemon` uses TCP port 50051 for gRPC connection.
Please check `calico-bgp-daemon` is listening on that port by
`docker exec calico-node ss -lt`